sionna/mapping.py

```diff
@@ -980,6 +980,8 @@ def __init__(self,
                                               dtype.real_dtype,
                                               **kwargs)
 
+        self._eps = np.finfo(dtype.as_numpy_dtype).eps
```

Comment:
I have two comments here:

1.) Please cast self._eps to tf.constant( ..., dtype._real_dtype)

2.) np.finfo(...).eps denotes the value of 1.0 and the smallest next number that can be represented. For float32 this is 1.1920929e-07 on my machine. Would it make sense to use np.finfo(...).tiny instead?

Reply:
1) Sure.
2) I originally thought that the input was normalized, so `eps` would make sense, but `tiny` is likely better, as you suggest; I will use a bigger value (`8*tiny`) to avoid getting into denormalized territory. Note that there are some other hardcoded "small number" constants over the codebase, so perhaps those should also be addressed.

Reply:
I would vote for using "just" `tiny` as it is more rigorous than `8*tiny` and the division happens directly afterwards (so there is no "denormalization territory" expected). But if there are good arguments for which `8*tiny` is beneficial, I see no real harm in using this version (given that `tiny` is so small that the impact on the LLRs is negligible).

Reply:
Done!